.github/workflows/review-check.yml

```diff
@@ -0,0 +1,56 @@
+name: Enforce Review Rules
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  enforce-review-rules:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Get list of files changed
+      id: files
+      run: |
+        echo "::set-output name=files::$(git diff --name-only ${{ github.event.before }} ${{ github.sha }} | tr '\n' ' ')"
```

Comment:
Echoing set-output to stdout is being [deprecated](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) and the syntax du jeur is writing to `$GITHUB_OUTPUT`.

Reply:
Fixed.